module/pteditor.h

```diff
@@ -61,8 +61,8 @@ typedef struct {
  */
 #if defined(LINUX)
 typedef struct {
-    /** Page-frame number */
-    size_t pfn;
+    /** Physical address */
+    size_t paddr;
```

Comment:
This will break backward compatibility as this is used in the user/kernel interface. Existing programs will stop working with the new version of the kernel module, which should not happen.